### DISCUSSION TITLE
Support `~/.ssh/config` and KeyChain for SFTP connectivity without exposing a password

### DISCUSSION THREAD

**Discussion by ww7** [mikekazakov/nimble-commander#353] [Ideas]
Many apps like VSCode, Forflift3 etc understands and respects `~/.ssh/config` where can be specified default IdentityFile and used with password from KeyChain

When `~/.ssh/config` consist
```
Host *
    addkeystoagent yes
    usekeychain yes
    IdentityFile ~/.ssh/key
```

Also would be super feature as quick connect that understands a hosts 
```
Host somename1
    HostName 77.55.0.1
Host somename2
    HostName 77.88.12.12   
```

Thanks!

**Comment by mikekazakov** [comment]
Yes, it would be a great addition, but somebody needs to actually go ahead and implement it. Without contributions this feature is very unlikely to appear in Nimble Commander.
